Argo.xcodeproj/project.pbxproj

```diff
@@ -441,6 +411,28 @@
 			path = Carthage/Build/Mac;
 			sourceTree = "<group>";
 		};
+		F87EB6981ABC5F1300E3B0AB /* Functions */ = {
+			isa = PBXGroup;
+			children = (
+				F87EB6991ABC5F1300E3B0AB /* curry.swift */,
+				F87EB69A1ABC5F1300E3B0AB /* decode.swift */,
+				F87EB69B1ABC5F1300E3B0AB /* flatReduce.swift */,
+				F87EB69C1ABC5F1300E3B0AB /* sequence.swift */,
+			);
+			path = Functions;
+			sourceTree = "<group>";
+		};
+		F87EB69D1ABC5F1300E3B0AB /* Types */ = {
+			isa = PBXGroup;
+			children = (
+				F87EB69E1ABC5F1300E3B0AB /* Decoded.swift */,
+				F87EB69F1ABC5F1300E3B0AB /* JSON.swift */,
+				F87EB6A01ABC5F1300E3B0AB /* JSONDecodable.swift */,
+				F87EB6A11ABC5F1300E3B0AB /* StandardTypes.swift */,
```

Comment:
What do you think about renaming this? Standard doesn't sounds right. Maybe Native?